degoo/__init__.py

```diff
@@ -1520,6 +1521,10 @@ def get_file(remote_file, local_directory=None, verbose=0, if_missing=False, dry
                     print(f"Would NOT download {Path}")
                 else:
                     print(f"Not downloading {Path}")
+    elif Data and Name:
+        decoded_content = base64.b64decode(Data).decode("utf-8")
+        with open(dest_file, "w") as text_file:
+            text_file.write(decoded_content)
```

Comment:
I take it from this that Degoo store small text files on their own servers and not on the Google Cloud Store, and don't deliver a URL in getOverlay3 but Data instead? I've not seen this. It's interesting.